docs/FAQ/how-deploy-bold-bi-in-eks-using-network-load-balancer-with-ssl-certificate-from-acm.md

```diff
@@ -0,0 +1,245 @@
+
+1. Download the [deploy.yaml](https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.5.1/deploy/static/provider/aws/nlb-with-tls-termination/deploy.yaml) template
+
+2. Edit the `deploy.yaml` file and change the VPC CIDR in use for the Kubernetes cluster.
+
+	```console
+	proxy-real-ip-cidr: XXX.XXX.XXX/XX
+	```
+	![VPC CIDR](images/VPC_CIDR.png)
+	
+	![Deploy VPC](images/deploy-vpc.png)
+	
+3. Change the AWS Certificate Manager (ACM) ID as well:
+
+	```console
+	arn:aws:acm:us-west-2:XXXXXXXX:certificate/XXXXXX-XXXXXXX-XXXXXXX-XXXXXXXX
+	```
+	![ACM CERT](images/acm-cert.png)
+	
+	![Deploy ACM](images/deploy-acm.png)
+
+4. Deploy the manifest:
+   
+   ```console
+   kubectl apply -f deploy.yaml
+   ```
+   
+5. Get the external IP and and map it with a DNS to craft values.yaml when installing Bold BI with helm chart.
+   
+   ```console
+   kubectl get svc -n ingress-nginx
+   ```
+   
+   ![NLB Ex_IP](images/nlb-ex-ip.png)
+   
+6. Check whether you can access the domain name with the secure protocol in browser.
+  
+   ![IP CHECK](images/ex-ip-checking.png)
+   
+	> **Note:** If you cannot access the URL in browser please ensure the below configuration.<br/><br/>
+		1. Ensure that the firewall or security group associated with the NLB allows incoming traffic on port 80 (HTTP) or 443 (HTTPS).<br/>
+		2. Make sure that the route to the NLB is correctly configured in your network infrastructure.<br/>
+		3. If the NLB is associated with a target group, ensure that the health check for the target group is healthy. The NLB will only forward traffic to healthy targets.<br/>
+		4. Ensure that the DNS for the NLB is correctly resolving to the correct IP address.<br/>
+		5. Ensure that the Kubernetes service associated with the NLB is correctly configured with the correct selector labels and target port.<br/>
+		6. Ensure that there are healthy pods running and associated with the Kubernetes service.<br/>
+
+## Get Repo Info
+
+1. Add the Bold BI helm repository.
+
+```console
+helm repo add boldbi https://boldbi.github.io/boldbi-kubernetes
+helm repo update
+```
+
+2. View charts in repo.
+
+```console
+helm search repo boldbi
+
+NAME            CHART VERSION   APP VERSION     DESCRIPTION
+boldbi/boldbi   5.3.83           5.3.83         Embed powerful analytics inside your apps and t...
+```
+
+_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+## Create Namespace
+
+Run the following command to create the namespace where the Bold BI resources will be deployed in the Kubernetes cluster. The default namespace is <i>bold-services</i>.
+
+```console
+kubectl create ns bold-services
+```
+
+## Install Chart
+
+For the Helm chart, you need to craft a `values.yaml` file. So download the values.yaml file from [here](https://raw.githubusercontent.com/boldbi/boldbi-kubernetes/UMP-22952-Autodeployment-documentation/helm/custom-values/eks-values.yaml) and make needed changes based on your cluster provider.
```

Comment:
For the Helm chart, you need to craft a `values.yaml` file. So, download the values.yaml file from [here](https://raw.githubusercontent.com/boldbi/boldbi-kubernetes/UMP-22952-Autodeployment-documentation/helm/custom-values/eks-values.yaml) and make the required changes based on your cluster provider.

Reply:
concern addressed